src/uvw/request.hpp

```diff
@@ -22,8 +22,8 @@ class Request: public Resource<T, U> {
         return ptr;
     }
 
-    template<typename R, typename E>
-    static void defaultCallback(R *req, int status) {
+    template<typename E>
+    static void defaultCallback(U *req, int status) {
```

Comment:
That's good. I introduced the `U` parameter during the second phase of the project and I forgot to remove this stuff. Nice catch.